policy/common/result.go

```diff
@@ -37,13 +37,22 @@ func (s EvaluationStatus) String() string {
 	return "unknown"
 }
 
+type AdminScope string
+
+const (
```

Comment:
Should we bail if an option provided doesn't match here?

Reply:
I think a sane default might be to assume `org` if unset? If you require admins to +1, but otherwise don't hint about what kind of admin, we should probably pick from anyone.